bin/slt-release.sh

```diff
@@ -78,6 +78,7 @@ slt-changelog --version "$V"
 
 echo "Updating package version to $V"
 slt version set "$V"
+git add $(git ls-files ../scripts/sl-blip.js)
```

Comment:
This is meant to be run from the package's root directory, so this would point at a path outside of the repo.